test/rules/invalid_case_patterns_test.dart

```diff
@@ -13,14 +13,11 @@ main() {
 }
 
 @reflectiveTest
-class InvalidCasePatternsTest extends LintRuleTest {
+class InvalidCasePatternsTest extends LintRuleTest
```

Comment:
Could you rename this to `InvalidCasePatternsTestLanguage219`?  That's the convention we've been converging on...

Thanks!

Reply:
Thanks, done!